charts/rasa-x/templates/rasa-services.yaml

```
@@ -7,6 +7,10 @@ metadata:
   labels:
     {{ include "rasa-x.labels" $ | nindent 4 }}
     app.kubernetes.io/component: {{ .serviceName }}
+{{- with .Values.rasa.service.annotations }}
```

The rasa service is a bit tricky
```suggestion
{{- with $.Values.rasa.service.annotations }}
```

---

Ha, I'd just figured this out 🙈  thanks!